docs/recipes/sass-and-nunjucks-with-static-asset-revisioning.md

```diff
@@ -0,0 +1,79 @@
+# Sass and Nunjucks with static asset revisioning
+
+Static asset revisioning by appending content hash to filenames. Make sure to set the files to [never expire](http://developer.yahoo.com/performance/rules.html#expires) for this to have an effect.
+
+Sass and Nunjucks are here just to demonstrate how to implement static asset revisioning in a more real-world scenario. The content has will be appended in production, so it's recommended to add the following npm scripts to `package.json`:
+
+```json
+{
+  "scripts": {
+    "dev": "gulp-dev",
+    "build": "NODE_ENV=production gulp build"
+  }
+}
+```
+
+Now to install the required dependencies:
+
+```sh
+npm install --save-dev gulp gulp-rev gulp-rev-rewrite gulp-sass gulp-nunjucks gulp-if del
+```
+
+And, finally, `gulpfile.js`:
+
+```js
+const gulp = require('gulp')
+const sass = require('gulp-sass')
+const nunjucks = require('gulp-nunjucks')
+const rev = require('gulp-rev')
+const revRewrite = require('gulp-rev-rewrite')
+const gulpIf = require('gulp-if')
+const del = require('del')
+
+const isProd = process.env.NODE_ENV === 'production'
+
+function clean() {
+  return del('dist')
+}
```

Comment:
If you are using static revisioning, I'm pretty sure you're never supposed to remove your old files (in case someone loads an old version from their cache).

Reply:
Oh… I thought that if an old version is being loaded, then static revisioning isn't set up correctly because the cache was supposed to be busted. That's a terrifying idea to me as an author that someone loads an old CSS or JS file, I guess things will break for them either way? 🤔

This clean task was not only for deleting old assets, but deleting old anything. Old pages etc. I want a 404 page to appear when a page has been deleted.  But I'll remove this task since we're focusing on revisioning only here.